### ISSUE TITLE
docker image missing?

### ISSUE THREAD

**Issue by christian-heusel** [OpenSlides/OpenSlides#5883]
Let me preface this with a big and warm thank you for the work y'all have been putting into OpenSlides ❤️ 

**Describe the bug**
The command following command from the `README.rst` fails: 
```
# docker-compose up
```

**How to Reproduce**

1. `$ git clone https://github.com/OpenSlides/OpenSlides.git --recurse-submodules`
2. `$ cd docker`
3. create/modify secret files
4. `# ./build.sh`
5. `$ m4 docker-compose.yml.m4 > docker-compose.yml`
6. `# docker-compose up`

```
chris@x -> sudo docker-compose up
Pulling pgnode1 (openslides/openslides-repmgr:latest)...
ERROR: The image for the service you're trying to recreate has been removed. If you continue, volume data could be lost. Consider backing up your data before continuing.

Continue with the new image? [yN]
ERROR: pull access denied for openslides/openslides-repmgr, repository does not exist or may require 'docker login': denied: requested access to the resource is denied
```

**Expected behavior**
From the README: OpenSlides runs on https://localhost:8000/ (◀️ is it really `https://` tho?)

**System information**

- OpenSlides-Version: master
- OS: Debian Unstable (5.10.0-3-amd64)
- `docker-compose --version`: `docker-compose version 1.25.0, build unknown`

**Additional context**
I am very happy to help & create a PR but would need pointer to where I can start with this issue 😊

**Comment by FinnStutzenstein** [comment]
Hi, thanks for the report!

I think you misread one step: It is `./build.sh all` instead of `./build.sh`. the `all` builds all images ;) If omitted, just the images of this repository are built (server, slient, autoupdate, proxy) and not external ones.

**Comment by tsiegleauq** [comment]
> is it really https:// tho?

yes. All communication relies on https.

**Comment by rasos** [comment]
I am having the same issue: 
```

Pulling pgnode1 (openslides/openslides-repmgr:latest)...
ERROR: The image for the service you're trying to recreate has been removed.
```

but I had applied `./build.sh all` and followed all steps as described in https://github.com/OpenSlides/OpenSlides#readme

Do I need to build extra what is in https://github.com/OpenSlides/openslides-docker-compose ?

**Comment by rasos** [comment]
It seems some dockers have not been built. Cloned and built all again. Now I did not apply `git submodule update --init` (which seems to be required only when upgrading form an older version). 

In `autoupdate/Docker` I had to set `golang:1.16.3-alpine3.12` (3.12.4 was not available).

Now all dockers have been built and start as expected. However Postgres cluster does not get available https://github.com/OpenSlides/OpenSlides/issues/5740#issuecomment-817197211
